app/src/main/java/org/p2p/wallet/send/model/SearchResult.kt

```diff
@@ -13,11 +14,16 @@ sealed class SearchResult(open val addressState: AddressState) : Parcelable {
     @Parcelize
     data class AddressOnly constructor(
         override val addressState: AddressState,
-        val sourceToken: Token.Active? = null
+        val sourceToken: Token.Active? = null,
+        val date: Date? = null
     ) : SearchResult(addressState)
 
     @Parcelize
-    data class UsernameFound(override val addressState: AddressState, val username: String) : SearchResult(addressState)
+    data class UsernameFound(
+        override val addressState: AddressState,
+        val username: String,
+        val date: Date? = null
```

Comment:
dateFound?

Reply:
it is date of Transaction so i prefer not to do something like dateFound

Reply:
i can't really couple transaction date and search result, but okay..
so date -> transactionDate?

Reply:
no
in context of search result there is no transactionDate